platform/src/main/java/org/hillview/storage/CassandraSSTableLoader.java

```diff
@@ -171,137 +230,269 @@ public Schema getSchema(CFMetaData metadata) {
+                                value = serializers.get(colType).deserialize(byteBuff);
+                                // Missing column case #2
+                                // Where loaded missing column(s) is still loaded and the capacity is 0
+                                if (byteBuff.capacity() == 0) {
+                                    col.appendMissing(); 
+                                } else {
+                                    switch ((CQL3Type.Native) colType) {
+                                        case ASCII:
+                                        case BOOLEAN:
+                                        case INET:
+                                        case TEXT:
+                                        case TIMEUUID:
+                                        case UUID:
+                                        case VARCHAR:
+                                            col.append(value.toString());
+                                            break;
+                                        case BLOB:
+                                            col.append(serializers.get(colType).toCQLLiteral(byteBuff).toString());
+                                            break;
+                                        case INT:
+                                            col.append((Integer) value);
+                                            break;
+                                        case SMALLINT:
+                                            col.append(((Short) value).intValue());
+                                            break;
+                                        case TINYINT:
+                                            col.append(((Byte) value).intValue());
+                                            break;
+                                        case VARINT:
+                                            col.append(((BigInteger) value).doubleValue());
+                                            break;
+                                        case BIGINT:
+                                            col.append(((Long) value).doubleValue());
+                                            break;
+                                        case DECIMAL:
+                                            if (bigDecimalInRange((BigDecimal) value))
+                                                col.append(((BigDecimal) value).doubleValue());
+                                            break;
+                                        case DOUBLE:
+                                            col.append((Double) value);
+                                            break;
+                                        case FLOAT:
+                                            col.append(((Float) value).doubleValue());
+                                            break;
+                                        case COUNTER:
+                                            col.append(CounterContext.instance().total(byteBuff));
```

Comment:
Isn't value already the long you need?

Reply:
Although the type is also Long, but Value isn't the right value. The (Long) Value gives me 422215120878763L, but the current value is 1L.